software/externals/cmake/externals.cmake

```diff
@@ -307,6 +307,29 @@ if(NOT APPLE)
 endif()
 
 
+if(USE_SYSTEM_OPENCV)
+  list(REMOVE_ITEM externals
+    opencv
+  )
+  list(REMOVE_ITEM occ-map_depends
+    opencv
+  )
+  list(REMOVE_ITEM libmultisense_depends
+    opencv
+  )
+endif()
```

Comment:
I guess doing list REMOVE_ITEM works fine, but did you see how I used the lcm_proj variable for the optional USE_SYSTEM_LCM?